asf_public_discourse_home_decarbonisation/analysis/buildhub/sentence_topic_and_sentiment_analysis_for_bh.py

```diff
@@ -0,0 +1,332 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+from asf_public_discourse_home_decarbonisation.getters.bh_getters import get_bh_data
+from asf_public_discourse_home_decarbonisation.utils.plotting_utils import (
+    plot_post_distribution_over_time,
+    plot_mentions_line_chart,
+)
+import pandas as pd
+from asf_public_discourse_home_decarbonisation.utils.text_processing_utils import (
+    process_abbreviations,
+)
+from asf_public_discourse_home_decarbonisation.utils.preprocessing_utils import (
+    preprocess_data_for_linechart_over_time,
+    resample_and_calculate_averages_for_linechart,
+)
+
+
+# In[2]:
+
+
+bh_data = get_bh_data(category="all")
```

Comment:
```suggestion
bh_data = get_bh_data(category="all", collection_date = "24_05_23"))
```
and then remove anything after the 22nd of june